
# to list from command: `ros2 param dump /map_analyzer --print`
map_analyzer:  # /map_analyzer or map_analyzer are both acceptable
  ros__parameters:
    local_radius: 200
    kernal_size: 15
    map_topic: 'map'
    odom_topic: 'odometry/filtered'

    use_sim_time: True

    # int_array:
    # - 10
    # - 11
    # str_array:
    # - 'str1'
    # - 'str2'

    # nested_param:
    #   another_byte_array:
    #   - 0x0A
    #   - 0x0B
    #   another_int: 101